fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2262,16 +2262,16 @@ The commit notification is sent in the request **PATCH /transfers/**_{ID}_ from
 
 **Figure 55 -- Commit notification where commit of transfer in Switch failed**
 
-In case of a validation failure (not schema validation) or another processing error during the processing of a **PUT /transfers/**_{ID}_ callback, a **PATCH /transfers/**_{ID}_**/error** notification is sent from the Switch to the Payee FSP to indicate of the failure. This notification is sent in both cases where the Payee FSP sends the _transferState_ as reserved or committed.
+In case of a validation failure (not schema validation) or another processing error during the processing of a **PUT /transfers/**_{ID}_ callback, a **PATCH /transfers/**_{ID}_**/error** notification can be sent from the Switch to the Payee FSP to indicate of the failure. This notification is sent in cases where the Payee FSP sends the _transferState_ as committed.
 
-[Figure 56](#figure-56) shows an example in which the validation in the Switch failed due to some reason, for example the a business rule validation caused a failure in the Switch. This is the same example as in [Figure 53](#figure-53), but where no reconciliation is needed as the Payee FSP receives a failure notification before performing the actual transfer to the Payee.
+[Figure 56a](#figure-56a) shows an example in which the validation in the Switch failed due to some reason, for example the a business rule validation caused a failure in the Switch. This is the same example as in [Figure 53](#figure-53), but the Payee FSP receives a failure notification before performing the actual transfer to the Payee so that the Payee FSP can take remedial action and helps decrease the need for reconciliation between the Switch and the Payee FSP.
```

Comment:
> , ... but the Payee FSP receives a failure notification before performing the actual transfer to the Payee ...

Error notifications should only be sent in case there is an actual error, so a Payee FSP can't wait to receive one as there is no guarantee that one will be sent. The Payee FSP shouldn't be allowed to wait with performing the transaction if transferState=COMMITTED. The Payee FSP should wait with committing the transaction if transferState=RESERVED is used, in this case a commit notification (PATCH /transfers/{ID}) is used either if the transfer was committed or failed in the Switch.